lib/spotlight/dor/indexer.rb

```diff
@@ -113,59 +156,52 @@ def add_folder_name(sdb, solr_doc)
-    # TODO:  push this up to stanford-mods gem?  or should it be hierarchical series/box/folder?
-    def add_series(sdb, solr_doc)
-      # see spec for data from actual collections
-      #   _location.physicalLocation should find top level and relatedItem
-      series_num = sdb.smods_rec._location.physicalLocation.map do |node|
-        val = node.text
-        # feigenbaum uses 'Accession'
-        match_data = val.match(/(?:(?:Series)|(?:Accession)):? ([^,|]+)/i)
-        match_data[1].strip if match_data.present?
-      end.compact
-
-      solr_doc['series_ssi'] = series_num.first
-    end
+    concerning :CartographicIndexing do
+      def mods_cartographics_indexing(sdb, solr_doc)
+        coordinates = Array(sdb.smods_rec.subject.cartographics.coordinates)
 
-    # rubocop:disable Metrics/AbcSize
-    def mods_cartographics_indexing sdb, solr_doc
-      insert_field(solr_doc, "coordinates", Array(sdb.smods_rec.subject.cartographics.coordinates).map { |n| n.text }, :stored_searchable)
+        insert_field(solr_doc, 'coordinates', coordinates.map(&:text), :stored_searchable)
 
-      Array(sdb.smods_rec.subject.cartographics.coordinates).map do |n|
-        next unless n.text =~ /^\(/ and n.text =~ /\)$/
+        solr_doc['point_bbox'] ||= []
+        solr_doc['point_bbox'] += coords_to_bboxes(coordinates)
+      end
 
-        bbox = n.text.gsub(/[\(\)]/, '')
+      private
 
-        lng, lat = bbox.split('/')
+      def coords_to_bboxes(coordinates)
+        coordinates.select { |n| n.text =~ /^\(.*\)$/ }.map do |n|
+          coord_to_bbox(n.text)
+        end
+      end
+
+      def coord_to_bbox(coord)
+        bbox = coord.delete('(').delete(')')
 
-        minX,maxX = lng.split('--').map { |x| coord_to_decimal(x) }
-        maxY,minY = lat.split('--').map { |x| coord_to_decimal(x) }
+        lng, lat = bbox.split('/')
 
-        solr_doc["point_bbox"] ||= []
-        solr_doc["point_bbox"] << "#{minX} #{minY} #{maxX} #{maxY}"
+        min_x, max_x = lng.split('--').map { |x| coord_to_decimal(x) }
+        max_y, min_y = lat.split('--').map { |y| coord_to_decimal(y) }
+        "#{min_x} #{min_y} #{max_x} #{max_y}"
       end
-    end
-    # rubocop:enable Metrics/AbcSize
 
-    def coord_to_decimal point
-      regex = /(?<dir>[NESW])\s*(?<deg>\d+)°(?:(?<sec>\d+)ʹ)?/
-      match = regex.match(point)
-      dec = 0
+      def coord_to_decimal(point)
+        regex = /(?<dir>[NESW])\s*(?<deg>\d+)°(?:(?<sec>\d+)ʹ)?/
+        match = regex.match(point)
+        dec = 0
 
-      dec += match['deg'].to_i
-      dec += match['sec'].to_f / 60
-      dec = -1 * dec if match['dir'] == "W" or match['dir'] == "S"
+        dec += match['deg'].to_i
+        dec += match['sec'].to_f / 60
+        dec = -1 * dec if match['dir'] == 'W' || match['dir'] == 'S'
 
-      dec
+        dec
+      end
     end
 
-    def insert_field solr_doc, field, values, *args
+    def insert_field(solr_doc, field, values, *args)
```

Comment:
i thought this was silly until i noticed it should really be named insert_field_values   b/c it's allowing multiple values to be inserted in one call.  maybe should have a comment too.  Bleah.